enterprise/frontend/src/metabase-enterprise/google_drive/GdriveConnectionModal.tsx

```diff
@@ -285,7 +266,11 @@ function GoogleSheetsDisconnectModal({
             {errorMessage}
           </Text>
           <Flex justify="flex-end" gap="md">
-            <Button variant="outline" onClick={onClose}>
+            <Button
+              variant="outline"
+              onClick={onClose}
+              disabled={isDeletingFolderLink}
```

Comment:
It felt weird before to click on the "Disconnect" and to still be able to click "Keep connected" (which couldn't stop the drive from getting disconnected).